cpf_test.go

```diff
@@ -0,0 +1,42 @@
+package godocs_test
+
+import (
+	"github.com/brazanation/go-documents"
+	"testing"
+)
+
+func TestShouldFormatCpf(t *testing.T) {
+	t.Run("CPF 06843273173", func(t *testing.T) {
+		cpf, err := godocs.NewCpf("06843273173")
+		if err == nil && cpf.Format() != "068.432.731-73" {
+			t.Error("CPF was not formatted")
+		}
+	})
+}
+
+func TestShouldThrowExceptionWithRepeatedNumbers(t *testing.T) {
```

Comment:
Is better to test it with a an array of testcases on `TestShouldThrowExceptionWithInvalidCpf`.
Have no diference on the behavior of `godocs .isValid` for repetead numbers or invalid number.